source/templates/conditionals.md

```diff
@@ -1,7 +1,49 @@
-Sometimes you may only want to display part of your template if a property
-exists.
+Conditional statements bring a minimum of logic into Ember templating.
+Statements like `if` and `unless` are implemented as built-in helpers. Helpers
+can be invoked three ways.
```

Comment:
Cool, I'm on board. I think I just approached it with a certain mindframe and the flow didn't fit into my mindframe, but taking a fresh look I think it works the way you've done it here.